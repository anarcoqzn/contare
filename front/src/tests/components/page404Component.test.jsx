import Page404Component from '../../components/page404Component/page404Component';

it("should render correctly", () =>{
    const wrapper = shallow(
        <Page404Component />
    );
    expect(wrapper).toMatchSnapshot();
});

