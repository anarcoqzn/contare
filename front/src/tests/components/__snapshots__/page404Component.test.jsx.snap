// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <h1>
    Página não Encontrada!
  </h1>
</div>
`;
